src/test/java/com/epam/tc/hw3/page/objects/composite/components/BottomPartComponent.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw3.page.objects.composite.components;
+
+import com.epam.tc.hw3.page.objects.composite.AbstractBaseComponent;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class BottomPartComponent extends AbstractBaseComponent {
+
+
+    @FindBy(css = "div.row.clerafix.benefits > div.col-sm-3 > "
+        + "div.benefit > div.benefit-icon > span.icons-benefit")
```

Comment:
I know that locator can be way more simple. That one and the same are too weak and depend on lots of classes and DOM structure.

Reply:
Changed to `span.icons-benefit`